orchestration/api/instances.py

```diff
@@ -11,3 +11,65 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from flask import jsonify
+from flask import Blueprint
+from orchestration.connectionmanager.Connector import Connector
+from flask import request
+import json
+from orchestration.db.api import create_workflow, create_workflow_definition
+
+instance = Blueprint("instance", __name__)
+
+# API to execute a service.
+# For example, St2 execution of this API will require 'action'
+# as one of the parameter. 'parameters' maybe required for the Actions
+# St2 example of data:/v1/orchestration/instances -H
+# "Content-Type:application/json" -d
+# '{"action":"aks.ak_echo_py", "parameters":{"message":"Hello World"}}'
+@instance.route("/v1/orchestration/instances", methods=['POST'])
```

Comment:
All our other APIs are still at v1beta.  This one should also be v1beta.

Reply:
changed to v1beta